src/model/kaniBinaryRunner.ts

```diff
@@ -181,13 +177,38 @@ async function captureFailedChecksForProof(
 // Run a command and capture the command line output into a string
 async function catchOutput(command: string, cargoKaniMode: boolean = false): Promise<number> {
 	const process = await execLog(command);
+	const process2 = await execLog2(command);
+	console.log(process2);
 	return process;
 }
 
+// exectute the command as a command line argument
+async function execLog2(command: string, cargoKaniMode: boolean = false): Promise<any> {
+
+	getKaniPath()
+	.then((cargoPath: any) => {
+		console.log(`Cargo path: ${cargoPath}`);
```

Comment:
It looks like this will be the second time this message is printed? First time is here 🧲